### DISCUSSION TITLE
How would I host mine without 7m.pl/5m.pl?

### DISCUSSION THREAD

**Discussion by Kingminer7** [Cvolton/GMDprivateServer#928] [Q&A]
I want to create a GDPS without paying, but 7m and 5m won't work. Are there any other places to host it?

**Comment by crypton1871** [comment]
1. 000webhost 
2. CrystalCloud
3. Bro literally any webhost with PHP, MySQL & subdomain

**Reply by Kingminer7** [reply]
000Webhost:
Doesn't the wiki say "Exact letter count required" for the link? Anything I try is too long.
CrystalCloud: 
Site will not load. Antivirus turned off and it still didn't work.

**Reply by crypton1871** [reply]
@Kingminer7 
> 000Webhost: Doesn't the wiki say "Exact letter count required" for the link? Anything I try is too long. CrystalCloud: Site will not load. Antivirus turned off and it still didn't work.

bro just rename the folder, it worked for me 

i have a gdps hosted on honeyx.000webhostapp.com/t

**Comment by ghost** [comment]
If you have a random old PC you don't use, you can install Linux and setup a GDPS on there (you need access to your router's settings to port forward and you will need a domain) I recommend [NoIP](https://noip.com/) as a free DDNS service but you will need to refresh the domain every 30ish days
